packages/rum/src/domain/getSessionReplayLink.spec.ts

```diff
@@ -5,7 +5,7 @@ import { getSessionReplayLink } from './getSessionReplayLink'
 import { addRecord, resetReplayStats } from './replayStats'
 
 const DEFAULT_CONFIGURATION = {
-  site: 'datad0g.com',
+  site: 'datadoghq.com',
```

Comment:
`datad0g.com` don't satisfies `Site`, We could cast it as `Site` but I don't think there is any reason not to use the production site in the unit tests